tests/e2e/upgrade/wdio.conf.js

```diff
@@ -118,11 +118,11 @@ const servicesStartTimeout = () => {
 
 // Override specific properties from wdio base config
 const upgradeConfig = Object.assign(wdioBaseConfig.config, {
-  specs: 
-   [
-     'upgrade.wdio-spec.js',
-     '*.wdio-spec.js'
-   ],
+  specs:
+    [
+      'upgrade.wdio-spec.js',
+      //'*.wdio-spec.js'
```

Comment:
Was this on purpose?

Reply:
Just trying to get a screenshot of the actual fail step.

Reply:
I will revert this.